ortho_seq_code/tests/test_orthoseqs.py

```diff
@@ -42,11 +34,23 @@ def test_cli(protein_seqs_no_padding, protein_pheno_no_padding):
     assert result.exit_code == 0
 
 
+def param_test(seqf, s, p, d):
+    with open(seqf) as f:
```

Comment:
we can make this a function as I saw that you repeated in the program itself as well as test

Reply:
I see what you mean. I don't know how to call the sites, pop_size, and dm variables from orthogonal_polynomial, so I redid them here. Any way I can address this?

Reply:
you have to define a function like this usually in a module separately called utils.py, you can call it in the tests as well as ortho_seqs module then.

def get_dm_pop_size_sites(seqf):
       # does something
       return dm, pop_size, sites